apps/test-bundles/webpack.config.js

```diff
@@ -0,0 +1,74 @@
+const path = require('path');
+const fs = require('fs');
+const resources = require('../../scripts/tasks/webpack-resources');
+
+// Files which should not be considered top-level entries.
+const TopLevelEntryFileExclusions = ['index.js', 'version.js', 'index.bundle.js'];
+const StatsFileName = 'test-bundles';
+
+module.exports = resources.createConfig(
+  StatsFileName,
+  true,
+  {
+    entry: _buildEntries('office-ui-fabric-react'),
+    externals: {
+      react: 'React',
+      'react-dom': 'ReactDOM'
+    },
+    plugins: [
+      {
+        apply: compiler => {
+          compiler.hooks.afterEmit.tap('AfterEmitPlugin', compilation => _copyStats('office-ui-fabric-react'));
+        }
+      }
+    ]
+  },
+  true,
+  true
+);
+
+/**
+ * Copy the stats file to dist folder the package the bundles were generated from.
+ */
+function _copyStats(packageName) {
+  try {
+    fs.copyFileSync(
+      path.join(__dirname, 'dist/test-bundles.stats.html'),
+      path.resolve(__dirname, `../../packages/${packageName}/dist/${StatsFileName}.stats.html`)
+    );
+  } catch (e) {
+    console.log(e);
+  }
+}
+
+/**
+ * Build webpack entries based on top level imports available in a package.
+ */
+function _buildEntries(packageName) {
```

Comment:
Is this something we place inside scripts/ ?

Reply:
potentially. getting it off the ground as a v1. Maybe this should be a separated build step?

Reply:
also it's smelly that i'm copying the stats html file to the dist folder.

And also it is not accessible through PR deploy. Is there any way we can alter that?